include/ttmlir/Dialect/TTNN/Utils/OptimizerOverrides.h

```diff
@@ -50,6 +80,89 @@ struct InputLayoutOverrideParser
                     const llvm::StringMap<InputLayoutOverrideParams> &value);
 };
 
+class OptimizerOverridesHandler {
+public:
+  OptimizerOverridesHandler() {};
+  ~OptimizerOverridesHandler() {};
+
+  // Setters for the overrides
+  // These are used to enable/disable the optimizer passes
+  void setOptimizerPass(bool);
+  // These are used to enable/disable the memory configurations
+  void setMemoryConfig(bool);
+  void setMemoryLayoutAnalysis(bool);
+  void setEnableMemoryLayoutAnalysisPolicy(bool);
```

Comment:
These names are a bit confusing to me, can we have them named exactly the same as their option strings? E.g. `setOptimizerPass` -> `setEnableOptimizer`?

Reply:
done

Reply:
Thanks. Can you make the same change for the rest of the setters in this comment?

Reply:
Done.